utils/rag/base_components.py

```diff
@@ -121,40 +127,41 @@ def init_embeddings(self):
 
         self.embeddings = SambaStudioEmbeddings()
     
-    def _display_image(self, image_bytes: bytes, width=512):
+    def _display_image(self, image_bytes: bytes, width: int=512) -> None:
         """
         Displays an image from a byte string.
 
         Args:
-            image_bytes (bytes): The byte string representing the image.
-            width (int, optional): The width of the displayed image. Defaults to 512.
+            image_bytes: The byte string representing the image.
+            width: The width of the displayed image. Defaults to 512.
 
         Returns:
-            The displayed image, via HTML.
+            None
         """
 
         decoded_img_bytes = base64.b64encode(image_bytes).decode('utf-8')
         html = f'<img src="data:image/png;base64,{decoded_img_bytes}" style="width: {width}px;" />'
         display(HTML(html))
     
-    def display_graph(self, app):
+    def display_graph(self, app: object) -> None:
         """
-        Displays a graph using Mermaid.
+        Prepares img_bytes of a graph using Mermaid for display purposes.  
+        Will be passed to the _display_image method for actual display.
 
         This method uses the Mermaid library to generate a graph and then displays it.
 
         Args:
             app: The application object that contains the graph to be displayed.
 
         Returns:
-            The populated self._display_image attribute.
+            None
         """
 
         nest_asyncio.apply() # Required for Jupyter Notebook to run async functions
 
         img_bytes = app.get_graph().draw_mermaid_png(
         curve_style=CurveStyle.LINEAR,
-        node_colors=NodeColors(start="#ffdfba", end="#baffc9", other="#fad7de"),
+        # node_colors=NodeColors(start="#ffdfba", end="#baffc9", other="#fad7de"),
```

Comment:
Could you add a comment for this commented line?

Reply:
I'll remove it.  Differences between LangGraph versions.  :/